README.md

```diff
@@ -41,21 +46,22 @@ One of the goals of this library is to make sure that the it is easy to use.  Ev
 **Velaptor** has the ability to load texture atlas image and JSON data for texture atlas type rendering as well as single image content and sound content.  The content loading system is flexible and has an API that has the ability to be extended to load custom content for your media applications and games.
 
 ### Content Caching
-Loaded content such as images and sounds are cached for reuse and performance reasons.  If the same content is loaded from disk after it has already been loaded, the same image content will be used for rendering.  This is the same for sounds and texture atlas data.  Custom content loading will not come with caching and will have to be added.
+Loaded content such as images and sounds are cached for performance reasons.  If the same content attempting to be loaded from disk after it has already been loaded, that same content will be used from memory instead of being reloaded.  This is the same for sounds and texture atlas data.  Custom content loading will not come with caching and will have to be added manually.
 
 ### Feature Demo Application
 https://user-images.githubusercontent.com/85414302/150527337-6d872768-73dc-4603-82e8-37c691c78d4d.mp4
 
-<h2 style="font-weight:bold" align="center">🔧Maintainers</h2>
+<h2 style="font-weight:bold;border:0" align="center">🔧Maintainers</h2>
 
 We currently have the following maintainers:
-- [Calvin Wilkinson](https://twitter.com/KDCoder) [<img src="https://about.twitter.com/etc/designs/about2-twitter/public/img/favicon.ico" alt="Follow Calvin Wilkinson on Twitter" width="16" />](https://twitter.com/KDCoder) (GitHub Organization/Project Owner)
+- [Calvin Wilkinson](https://twitter.com/KDCoder) [<img src="https://about.twitter.com/etc/designs/about2-twitter/public/img/favicon.ico" alt="Follow Calvin Wilkinson on Twitter" width="16" />](https://twitter.com/KDCoder) (GitHub Organization / Owner)
+- [Kristen Wilkinson](https://twitter.com/kswilky) [<img src="https://about.twitter.com/etc/designs/about2-twitter/public/img/favicon.ico" alt="Follow Calvin Wilkinson on Twitter" width="16" />](https://twitter.com/KDCoder) (GitHub Organization / Documentation Maintainer / Tester)
 
-<h2 style="font-weight:bold" align="center">📄Documentation</h2>
+<h2 style="font-weight:bold;border:0" align="center">📄Documentation</h2>
 
 - Goto the [Table Of Contents](./Documentation/TableOfContents.md) for instructions on various things such as the branching, release process, environment setup and more.
 
-<h2 style="font-weight:bold" align="center">🙏🏼Contributing</h2>
+<h2 style="font-weight:bold;border:0" align="center">🙏🏼Contributing</h2>
 
 **Velaptor** encourages and uses [Early Pull Requests](https://medium.com/practical-blend/pull-request-first-f6bb667a9b6). Please don't wait until you're finished with your work before creating a PR.
 
```

Comment:
Make sure you put periods after all of the sentences. 
1. Fork the repository.
7.  Refer to the editorconfig files in the code base for rules.